data/distancing/functions/check_end_exits.mcfunction

```diff
@@ -9,5 +9,7 @@ setblock ~-1 64 0 bedrock
 setblock ~ 64 1 bedrock
 setblock ~ 64 -1 bedrock
 
-execute if entity @e[type=ender_dragon] run setblock ~ 64 0 air
-execute unless entity @e[type=ender_dragon] run setblock ~ 64 0 end_portal
+execute store success score $success dist_mem run clone -3 0 -3 3 255 3 -3 0 -3 filtered minecraft:end_portal move
```

Comment:
Why not just `if block`?

Reply:
The end portal does not have a fixed height, so you'd need to check at least one entire y column to be sure